test/etc/profile.definitions/site/EasyBuild.yml

```diff
@@ -0,0 +1,5 @@
+## help EasyBuild
+EASYBUILD:
+  BUILDPATH:     /dev/shm/$USER
```

Comment:
Whether this is sensible depends on a number of things (amount of available memory among one). Some builds will also fail because `/dev/shm/` is used as build directory (can't say I have a quick example though).

Reply:
necessary evil or, to put it otherwise, not doing it might be more evil (spammed parallel fs)

Reply:
@boegel : thanks for reviewing!